{"code":"ESOCKET","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","name":"ConnectionError","originalError":{"code":"ESOCKET"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: \n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.socketError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1353:12)\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:2258:25\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-05-15T13:06:00.724Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","service":"automacao-service","timestamp":"2025-05-15T13:06:00.725Z"}
{"code":"ESOCKET","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","name":"ConnectionError","originalError":{"code":"ESOCKET"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: \n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.socketError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1353:12)\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:2258:25\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-05-15T13:07:00.640Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","service":"automacao-service","timestamp":"2025-05-15T13:07:00.641Z"}
{"code":"ESOCKET","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","name":"ConnectionError","originalError":{"code":"ESOCKET"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: \n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.socketError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1353:12)\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:2258:25\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-05-15T13:08:00.673Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","service":"automacao-service","timestamp":"2025-05-15T13:08:00.674Z"}
{"code":"ESOCKET","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","name":"ConnectionError","originalError":{"code":"ESOCKET"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: \n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.socketError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1353:12)\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:2258:25\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-05-15T13:09:00.664Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","service":"automacao-service","timestamp":"2025-05-15T13:09:00.664Z"}
{"code":"ESOCKET","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","name":"ConnectionError","originalError":{"code":"ESOCKET"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: \n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.socketError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1353:12)\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:2258:25\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-05-15T13:10:00.648Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 - Hostname/IP does not match certificate's altnames: IP: 216.158.231.74 is not in the cert's list: ","service":"automacao-service","timestamp":"2025-05-15T13:10:00.649Z"}
{"class":16,"code":"EREQUEST","level":"error","lineNumber":2,"message":"Erro durante o processo de cobrança: Invalid object name 'Assinaturas'.","name":"RequestError","number":208,"originalError":{"info":{"class":16,"handlerName":"onErrorMessage","lineNumber":2,"message":"Invalid object name 'Assinaturas'.","name":"ERROR","number":208,"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","state":1}},"precedingErrors":[],"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","service":"automacao-service","stack":"RequestError: Invalid object name 'Assinaturas'.\n    at handleError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\request.js:384:15)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at RequestTokenHandler.onErrorMessage (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\handler.js:284:21)\n    at Readable.<anonymous> (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\token-stream-parser.js:19:33)\n    at Readable.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushObjectMode (node:internal/streams/readable:536:3)\n    at Readable.push (node:internal/streams/readable:391:5)\n    at nextAsync (node:internal/streams/from:194:22)","state":1,"timestamp":"2025-05-15T13:11:02.389Z"}
{"level":"error","message":"Job de cobrança falhou: Invalid object name 'Assinaturas'.","service":"automacao-service","timestamp":"2025-05-15T13:11:02.389Z"}
{"class":16,"code":"EREQUEST","level":"error","lineNumber":2,"message":"Erro durante o processo de cobrança: Invalid object name 'Assinaturas'.","name":"RequestError","number":208,"originalError":{"info":{"class":16,"handlerName":"onErrorMessage","lineNumber":2,"message":"Invalid object name 'Assinaturas'.","name":"ERROR","number":208,"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","state":1}},"precedingErrors":[],"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","service":"automacao-service","stack":"RequestError: Invalid object name 'Assinaturas'.\n    at handleError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\request.js:384:15)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at RequestTokenHandler.onErrorMessage (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\handler.js:284:21)\n    at Readable.<anonymous> (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\token-stream-parser.js:19:33)\n    at Readable.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushObjectMode (node:internal/streams/readable:536:3)\n    at Readable.push (node:internal/streams/readable:391:5)\n    at nextAsync (node:internal/streams/from:194:22)","state":1,"timestamp":"2025-05-15T13:12:02.186Z"}
{"level":"error","message":"Job de cobrança falhou: Invalid object name 'Assinaturas'.","service":"automacao-service","timestamp":"2025-05-15T13:12:02.187Z"}
{"class":16,"code":"EREQUEST","level":"error","lineNumber":2,"message":"Erro durante o processo de cobrança: Invalid object name 'Assinaturas'.","name":"RequestError","number":208,"originalError":{"info":{"class":16,"handlerName":"onErrorMessage","lineNumber":2,"message":"Invalid object name 'Assinaturas'.","name":"ERROR","number":208,"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","state":1}},"precedingErrors":[],"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","service":"automacao-service","stack":"RequestError: Invalid object name 'Assinaturas'.\n    at handleError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\request.js:384:15)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at RequestTokenHandler.onErrorMessage (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\handler.js:284:21)\n    at Readable.<anonymous> (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\token-stream-parser.js:19:33)\n    at Readable.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushObjectMode (node:internal/streams/readable:536:3)\n    at Readable.push (node:internal/streams/readable:391:5)\n    at nextAsync (node:internal/streams/from:194:22)","state":1,"timestamp":"2025-05-15T13:13:02.256Z"}
{"level":"error","message":"Job de cobrança falhou: Invalid object name 'Assinaturas'.","service":"automacao-service","timestamp":"2025-05-15T13:13:02.257Z"}
{"class":16,"code":"EREQUEST","level":"error","lineNumber":2,"message":"Erro durante o processo de cobrança: Invalid object name 'Assinaturas'.","name":"RequestError","number":208,"originalError":{"info":{"class":16,"handlerName":"onErrorMessage","lineNumber":2,"message":"Invalid object name 'Assinaturas'.","name":"ERROR","number":208,"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","state":1}},"precedingErrors":[],"procName":"","serverName":"Plesk7200\\MSSQLSERVER2022","service":"automacao-service","stack":"RequestError: Invalid object name 'Assinaturas'.\n    at handleError (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\request.js:384:15)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at RequestTokenHandler.onErrorMessage (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\handler.js:284:21)\n    at Readable.<anonymous> (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\token\\token-stream-parser.js:19:33)\n    at Readable.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushObjectMode (node:internal/streams/readable:536:3)\n    at Readable.push (node:internal/streams/readable:391:5)\n    at nextAsync (node:internal/streams/from:194:22)","state":1,"timestamp":"2025-05-15T13:14:02.255Z"}
{"level":"error","message":"Job de cobrança falhou: Invalid object name 'Assinaturas'.","service":"automacao-service","timestamp":"2025-05-15T13:14:02.255Z"}
{"level":"error","message":"Erro durante o processo de cobrança: Cannot read properties of null (reading 'includes')","service":"automacao-service","stack":"TypeError: Cannot read properties of null (reading 'includes')\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:17:91\n    at Array.filter (<anonymous>)\n    at CobrancaService.executarCobranca (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:17:54)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async cron.schedule.scheduled (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\jobs\\cobrancaJob.js:15:27)\n    at async C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\node-cron\\dist\\cjs\\scheduler\\runner.js:67:44","timestamp":"2025-05-15T15:54:02.677Z"}
{"level":"error","message":"Job de cobrança falhou: Cannot read properties of null (reading 'includes')","service":"automacao-service","timestamp":"2025-05-15T15:54:02.679Z"}
{"level":"error","message":"Erro durante o processo de cobrança: Cannot read properties of null (reading 'includes')","service":"automacao-service","stack":"TypeError: Cannot read properties of null (reading 'includes')\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:17:91\n    at Array.filter (<anonymous>)\n    at CobrancaService.executarCobranca (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:17:54)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async cron.schedule.scheduled (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\jobs\\cobrancaJob.js:15:27)\n    at async C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\node-cron\\dist\\cjs\\scheduler\\runner.js:67:44","timestamp":"2025-05-15T15:55:02.543Z"}
{"level":"error","message":"Job de cobrança falhou: Cannot read properties of null (reading 'includes')","service":"automacao-service","timestamp":"2025-05-15T15:55:02.544Z"}
{"level":"error","message":"Erro durante o processo de cobrança: sociosVeiculosAssinados is not defined","service":"automacao-service","stack":"ReferenceError: sociosVeiculosAssinados is not defined\n    at CobrancaService.executarCobranca (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:22:34)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async cron.schedule.scheduled (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\jobs\\cobrancaJob.js:15:27)\n    at async C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\node-cron\\dist\\cjs\\scheduler\\runner.js:67:44","timestamp":"2025-05-15T15:56:02.919Z"}
{"level":"error","message":"Job de cobrança falhou: sociosVeiculosAssinados is not defined","service":"automacao-service","timestamp":"2025-05-15T15:56:02.920Z"}
{"level":"error","message":"Erro durante o processo de cobrança: sociosVeiculosAssinados is not defined","service":"automacao-service","stack":"ReferenceError: sociosVeiculosAssinados is not defined\n    at CobrancaService.executarCobranca (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:22:34)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async cron.schedule.scheduled (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\jobs\\cobrancaJob.js:15:27)\n    at async C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\node-cron\\dist\\cjs\\scheduler\\runner.js:67:44","timestamp":"2025-05-15T15:58:02.635Z"}
{"level":"error","message":"Job de cobrança falhou: sociosVeiculosAssinados is not defined","service":"automacao-service","timestamp":"2025-05-15T15:58:02.636Z"}
{"level":"error","message":"Erro durante o processo de cobrança: sociosVeiculosAssinados is not defined","service":"automacao-service","stack":"ReferenceError: sociosVeiculosAssinados is not defined\n    at CobrancaService.executarCobranca (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:22:34)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async cron.schedule.scheduled (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\jobs\\cobrancaJob.js:15:27)\n    at async C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\node-cron\\dist\\cjs\\scheduler\\runner.js:67:44","timestamp":"2025-05-15T15:59:02.534Z"}
{"level":"error","message":"Job de cobrança falhou: sociosVeiculosAssinados is not defined","service":"automacao-service","timestamp":"2025-05-15T15:59:02.534Z"}
{"level":"error","message":"Erro durante o processo de cobrança: sociosVeiculosAssinados is not defined","service":"automacao-service","stack":"ReferenceError: sociosVeiculosAssinados is not defined\n    at CobrancaService.executarCobranca (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\services\\cobrancaService.js:22:34)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async cron.schedule.scheduled (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\src\\jobs\\cobrancaJob.js:15:27)\n    at async C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\node-cron\\dist\\cjs\\scheduler\\runner.js:67:44","timestamp":"2025-05-15T16:00:02.612Z"}
{"level":"error","message":"Job de cobrança falhou: sociosVeiculosAssinados is not defined","service":"automacao-service","timestamp":"2025-05-15T16:00:02.613Z"}
{"code":"ETIMEOUT","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 in 15000ms","name":"ConnectionError","originalError":{"code":"ETIMEOUT"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 in 15000ms\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.connectTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1222:10)\n    at Timeout._onTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1167:12)\n    at listOnTimeout (node:internal/timers:581:17)\n    at process.processTimers (node:internal/timers:519:7)","timestamp":"2025-05-15T16:01:15.099Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 in 15000ms","service":"automacao-service","timestamp":"2025-05-15T16:01:15.100Z"}
{"code":"ETIMEOUT","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 in 15000ms","name":"ConnectionError","originalError":{"code":"ETIMEOUT"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 in 15000ms\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.connectTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1222:10)\n    at Timeout._onTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1167:12)\n    at listOnTimeout (node:internal/timers:581:17)\n    at process.processTimers (node:internal/timers:519:7)","timestamp":"2025-05-15T16:11:15.915Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 in 15000ms","service":"automacao-service","timestamp":"2025-05-15T16:11:15.915Z"}
{"code":"ETIMEOUT","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 in 15000ms","name":"ConnectionError","originalError":{"code":"ETIMEOUT"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 in 15000ms\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.connectTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1222:10)\n    at Timeout._onTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1167:12)\n    at listOnTimeout (node:internal/timers:581:17)\n    at process.processTimers (node:internal/timers:519:7)","timestamp":"2025-05-26T21:32:15.053Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 in 15000ms","service":"automacao-service","timestamp":"2025-05-26T21:32:15.054Z"}
{"code":"ETIMEOUT","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 in 15000ms","name":"ConnectionError","originalError":{"code":"ETIMEOUT"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 in 15000ms\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.connectTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1222:10)\n    at Timeout._onTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1167:12)\n    at listOnTimeout (node:internal/timers:581:17)\n    at process.processTimers (node:internal/timers:519:7)","timestamp":"2025-05-26T21:33:15.912Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 in 15000ms","service":"automacao-service","timestamp":"2025-05-26T21:33:15.912Z"}
{"code":"ETIMEOUT","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 in 15000ms","name":"ConnectionError","originalError":{"code":"ETIMEOUT"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 in 15000ms\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.connectTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1222:10)\n    at Timeout._onTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1167:12)\n    at listOnTimeout (node:internal/timers:581:17)\n    at process.processTimers (node:internal/timers:519:7)","timestamp":"2025-05-26T21:35:15.045Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 in 15000ms","service":"automacao-service","timestamp":"2025-05-26T21:35:15.046Z"}
{"code":"ETIMEOUT","level":"error","message":"Erro durante o processo de cobrança: Failed to connect to 216.158.231.74:1433 in 15000ms","name":"ConnectionError","originalError":{"code":"ETIMEOUT"},"service":"automacao-service","stack":"ConnectionError: Failed to connect to 216.158.231.74:1433 in 15000ms\n    at C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\mssql\\lib\\tedious\\connection-pool.js:85:17\n    at Connection.onConnect (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:849:9)\n    at Object.onceWrapper (node:events:634:26)\n    at Connection.emit (node:events:519:28)\n    at Connection.emit (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:970:18)\n    at Connection.connectTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1222:10)\n    at Timeout._onTimeout (C:\\Users\\marco\\OneDrive\\Documentos\\GitHub\\vcar-bot\\node_modules\\tedious\\lib\\connection.js:1167:12)\n    at listOnTimeout (node:internal/timers:581:17)\n    at process.processTimers (node:internal/timers:519:7)","timestamp":"2025-05-26T21:39:15.918Z"}
{"level":"error","message":"Job de cobrança falhou: Failed to connect to 216.158.231.74:1433 in 15000ms","service":"automacao-service","timestamp":"2025-05-26T21:39:15.918Z"}
